lib/jxl/enc_ac_strategy.cc

```diff
@@ -947,16 +952,28 @@ void ProcessRectACSNew(PassesEncoderState* JXL_RESTRICT enc_state,
     uint8_t priority;
     float entropy_mul;
   };
+  static const float k8x16mul1 = -0.49793408372209957;
+  static const float k8x16mul2 = 1.0265154587144558f;
+  static const float k8x16base = 2.1458510646431797f;
+  const float mul8x16 =
+      k8x16mul2 + k8x16mul1 / (butteraugli_target + k8x16base);
+
+  static const float k16x16mul1 = -0.068006033511844949;
+  static const float k16x16mul2 = 1.0113327118184485;
+  static const float k16x16base = 2.6559390179313955f;
+  const float mul16x16 =
+      k16x16mul2 + k16x16mul1 / (butteraugli_target + k16x16base);
+
   const MergeTry kTransformsForMerge[9] = {
-      {AcStrategy::Type::DCT16X8, 2, 0.86f},
-      {AcStrategy::Type::DCT8X16, 2, 0.86f},
-      {AcStrategy::Type::DCT16X16, 3, 0.83f},
-      {AcStrategy::Type::DCT16X32, 4, 0.94f},
-      {AcStrategy::Type::DCT32X16, 4, 0.94f},
-      {AcStrategy::Type::DCT32X32, 5, 0.97f},
-      {AcStrategy::Type::DCT64X32, 6, 1.15f},
-      {AcStrategy::Type::DCT32X64, 6, 1.15f},
-      {AcStrategy::Type::DCT64X64, 8, 1.3f},
+      {AcStrategy::Type::DCT16X8, 2, mul8x16 * 0.91195782912371126f},
+      {AcStrategy::Type::DCT8X16, 2, mul8x16 * 0.91195782912371126f},
+      {AcStrategy::Type::DCT16X16, 3, mul16x16 * 0.83183417727960129f},
+      {AcStrategy::Type::DCT16X32, 4, 0.88854513227338527f},
+      {AcStrategy::Type::DCT32X16, 4, 0.88854513227338527f},
+      {AcStrategy::Type::DCT32X32, 5, 1.0092994906548809f},
+      {AcStrategy::Type::DCT64X32, 6, 2.0858810264509633f},
+      {AcStrategy::Type::DCT32X64, 6, 2.0858810264509633f},
+      {AcStrategy::Type::DCT64X64, 8, 2.0846542128012948f},
```

Comment:
These new 2.something multipliers for 32x64 and 64x64 make it a lot less likely to use those, even at high distance targets. Is that a good idea?

Reply:
optimization thinks like that right now, at least using my current benchmark material which does not have zillions of pixels of blue sky

Reply:
I'm adding a TODO to re-optimize these

Reply:
Also effectively when the multipliers for smaller blocks are below 1, this raises the bar for the bigger blocks even higher - in that sense these constants are not independent (e.g. changing the constant for DCT16x32 by -5% (making it more likely) also means that DCT32x32 becomes harder to do when starting from two DCT16x32s). It might be better to make them more independent, e.g. by not applying the multiplier when storing the new entropy estimates in TryMergeToACSCandidate().